nifi-commons/nifi-processor-utilities/src/test/java/org/apache/nifi/processor/util/TestStandardValidators.java

```diff
@@ -100,4 +100,105 @@ public void testDataSizeBoundsValidator() {
         vr = val.validate("DataSizeBounds", "water", validationContext);
         assertFalse(vr.isValid());
     }
+
+    @Test
+    public void testListValidator() {
+        Validator val = StandardValidators.createListValidator(true, false, StandardValidators.NON_EMPTY_VALIDATOR);
```

Comment:
The trimming behavior would probably be easier to test with the BOOLEAN_VALIDATOR or similar.

Ex:
```
vr = val.validate("List", "notbool", validationContext);
assertFalse(vr.isValid());

vr = val.validate("List", "true", validationContext);
assertTrue(vr.isValid());

vr = val.validate("List", "    true   \n   ", validationContext);
assertTrue(vr.isValid());
```